questions/qa-i18n.zh.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="zh-Has">
+<head>
+  <meta charset="utf-8" />
+  <title>本地化 vs 国际化</title>
+  <meta name="description" content="术语“国际化”和“本地化”是什么意思，它们之间有什么关系？" />
+  <script>
+    var f = { };
+    // AUTHORS should fill in these assignments:
+    f.directory = 'questions'+'/';
+    // the path to this file, not including /International or the file name
+    f.filename = 'qa-i18n';
+    // the file name WITHOUT extensions
+    f.authors = 'Richard Ishida, W3C, Susan K. Miller, Boeing';
+    // author(s) and affiliations
+    f.previousauthors = '';
+    // as above
+    f.modifiers = '';
+    // people making substantive changes, and their affiliation
+    f.searchString = 'qa-i18n';
+    // blog search string - usually the filename without extensions
+    f.firstPubDate = '2005-12-05';
+    // date of the first publication of the document (after review)
+    f.lastSubstUpdate = { date:'2005-12-05', time:'20:40'};
+    // date and time of latest substantive changes to this document
+    f.status = 'published';
+    // should be one of draft, review, published, notreviewed or obsolete
+    f.path = '../';
+    // what you need to prepend to a URL to get to the /International directory
+    // AUTHORS AND TRANSLATORS should fill in these assignments:
+    f.thisVersion = {date:'2020-09-11', time:'09:11'};
+    // date and time of latest edits to this document/translation
+    f.contributors = 'xgqfrms';
```

Comment:
No need for this value, it's for people helping with reviews, etc. You are credited just below as translator.